messages/cache_settings/errors.go

```diff
@@ -5,26 +5,26 @@ import (
 )
 
 var (
-    ErrorGetCaches          = errors.New("Failed to list your cache settings. Check your settings and try again. If the error persists, contact Azion support.")
-    ErrorGetCache           = errors.New("Failed to get cache settings: %s. Check your settings and try again. If the error persists, contact Azion support.")
-    ErrorMandatoryListFlags = errors.New("Required flags are missing. You must provide application-id, name, addresses and host-header flags when the --application-id flag are not provided. Run the command 'azioncli <command> <subcommand> --help' to display more information and try again.")
+    ErrorGetCaches          = errors.New("Failed to list your Cache Settings configurations. Check your settings and try again. If the error persists, contact Azion support.")
```

Comment:
Cache Settings will always be capitalized then?

Reply:
Having the [docs](https://www.azion.com/en/documentation/products/edge-application/edge-caching/#cache-settings) as reference, yes. That's the way they treat it.